docs/CHANGELOG.asciidoc

```diff
@@ -68,6 +68,440 @@ Review important information about the {kib} 7.x releases.
 
 coming::[7.15.0]
 
+For information about the {kib} 7.15.0 release, review the following information.
+
+[float]
+[[known-issue-7.15.0]]
+=== Known issue
+
+.Upgrade Elastic Agents to use Osquery Manager integration
+[%collapsible]
+====
+*Details* +
+You must upgrade your Elastic Agents to the latest version to use the Osquery Manager integration.
+
+*Impact* +
+To upgrade, refer to {fleet-guide}/upgrade-elastic-agent.html[Upgrade Elastic Agent].
+====
+
+[float]
+[[breaking-changes-7.15.0]]
+=== Breaking changes
+
+Breaking changes can prevent your application from optimal operation and performance.
+Before you upgrade to 7.15.0, review the breaking changes, then mitigate the impact to your application.
+
+// tag::notable-breaking-changes[]
+      
+[discrete]
+[[breaking-103652]]
+.Migrates machine learning URL generators to locators
+[%collapsible]
+====
+*Details* +
+In *Machine Learning*, *APM*, *Logs*, and *Security*, the machine learning URL generator has been replaced with a machine learning locator. For more information, refer to {kibana-pull}103652[#103652].
+
+*Impact* +
+When you upgrade, the URL generators are automatically migrated to use the new locators. 
+====
+
+[discrete]
+[[breaking-osquery]]
+.New scheduled query action ID format for Osquery Manager
+[%collapsible]
+====
+*Details* +
+Action IDs for scheduled queries have been changed, which may break dashboards, alerts, or other features that depend on Osquery action IDs.
+
+*Impact* +
+Previously, scheduled query action IDs used the query ID. For example, if you scheduled a query with the name `rpm_packages`, the action ID would be `rpm_packages`. Now, scheduled query action IDs use the `pack_<scheduled query group name>_<query ID>` format. For example, if you have a scheduled query with the name `rpm_packages` in a group named `system_monitoring`, the action ID would be `pack_system_monitoring_rpm_packages`.
+====
+
+[discrete]
+[[deprecation-105055]]
+.Removes support for legacy charts library
+[%collapsible]
+====
+*Details* +
+The legacy area, line, bar, and pie charts library are deprecated in 7.15.0. In 7.16.0, support for the legacy charts library will be removed. For more information, refer to {kibana-pull}105055[#105055].
```

Comment:
In 7.16.0, only xy chart legacy support will be removed. pie will still be supported. This has to be changed in this sentence and in the impact section.

```
In 7.16.0, support for the area, line and bar legacy charts library will be removed
```
```
You must use the new charts library to create area, line, and bar charts, which includes improved...
```

Reply:
We should also make it clear that end users don't need to do anything unless they have overwritten the default settings. Right now it sounds like everyone needs to fix their deployments